TrkFilters/src/HelixFilter_module.cc

```diff
@@ -191,6 +201,61 @@ namespace mu2e
     }
     return true;
   }
+
+
+  float   HelixFilter::helixHitRatio(const HelixSeed* helix){
+
+    int      nCH = helix->hits().size();
+    int      nFaces(36); //was 36, now doing stations instead of faces
+    float    expected_hits(0);
+    float    z_step(178.); //are they really equally spaced? was 89 before
+    float    minR_tracker(380.);
+    float    maxR_tracker(860.);
```

Comment:
The active volume of the tracker extends to 700, did you mean 680 here?

Reply:
typo... DONE

Reply:
Hi Dave et al,
I tried to get the correct numbers from the "innerTrackerEnvelopeParams" data member of the Tracker class, but it shows the following (see attached picture).
How can I get the actaul outerRadius of the active volume?
Thanks,

Giani
```suggestion
    float    maxR_tracker(860.);
<img width="178" alt="Screen Shot 2019-12-18 at 11 30 30 PM" src="https://user-images.githubusercontent.com/25443728/71132368-8463e780-21f7-11ea-8413-8fd696b0aeec.png">

```